SPQR/SPQRGPU/spqrgpu_computeFrontStaging.cpp

```diff
@@ -237,84 +237,27 @@ void spqrgpu_computeFrontStaging
     return;
 }
 
-template void spqrgpu_computeFrontStaging
```

Comment:
I can't have these comments removed.

Reply:
They are still here: https://github.com/mmuetzel/SuiteSparse/blob/112289734be3f6de6ef8aa3bb5eeb701d0612fc9/SPQR/Include/spqrgpu.hpp#L28

They've just very recently been copied to this file. Imho, having to search through all files in case you'd like to update the comments for some reason will be hard to remember...